Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md

```diff
@@ -0,0 +1,55 @@
+This playbook is designed to handle the following alerts:  
+- Possible LDAP enumeration by unsigned process (medium severity)
```

Comment:
```suggestion
- Possible LDAP enumeration by an unsigned process (medium severity)
```

Reply:
If the mistake is in the alert name we need to keep it as it is in the alert name